src/pages/Landing/LandingPage.tsx

```diff
@@ -45,6 +45,7 @@ class LandingPage extends React.Component<LandingPageProps> {
                         }}
                         type="fadeIn">
                         {/* TODO Route zurück auf /login /register */}
+                        <Register />
```

Comment:
register wasn't visible for me. Probably that was my fault, but verify that the route register shows also the modal (change the value in the store)